src/Models/Sql/SqlRecommendation.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace AzureMcp.Models.Sql;
+
+/// <summary>
+/// Represents a recommendation for a SQL database from Azure SQL Database advisors.
+/// </summary>
+public sealed record SqlRecommendation
```

Comment:
Why do we need to re-expose return/result types vs leveraging ones directly from the SDKs? We're returning JSON anyway - so why do we need all these boilerplate types?